internal/distributed/indexnode/client/client.go

```diff
@@ -61,10 +62,12 @@ func (c *Client) Init() error {
 }
 
 func (c *Client) connect(retryOptions ...retry.Option) error {
+	ctx, cancel := context.WithTimeout(c.ctx, 10*time.Second)
```

Comment:
I don't recommend to do so. This may cause the chain broken, all the trace and timeout not working.

Why not refactor the part and do it right?

Reply:
Discussed with @czs007, all nodes have this problem, @godchen0212  is refactoring this part of the code. This pr can be closed.

Reply:
pr: #7203